docs/generateBuildMatrix.sh

```diff
@@ -2,39 +2,51 @@
 
 # Generates markdown table of build job status
 
-if [[ -f "/tmp/build.txt" ]]; then
-  echo "Removing previous /tmp/build.txt /tmp/build_jobs.txt files"
-  rm "/tmp/build.txt"
-  rm "/tmp/build_jobs.txt"
+allPlatforms=("jdk8u" "jdk11u" "jdk17u" "jdk18u" "jdk")
+buidFile="/tmp/build.txt"
```

Comment:
looks reasonable, just wondering if "buidFile", should be "buildFile"? it's consistent throughout but wondering about spelling?

Reply:
You are right, it was a typo, will fix in next commit